model/cairoItems.cc

```diff
@@ -94,7 +94,7 @@ bool RenderVariable::inImage(float x, float y)
 
 double RenderVariable::handlePos() const
 {
-  var.initSliderBounds();
+  if (var.sliderStep<1.0e-100) var.initSliderBounds();   // this should only be used when sliderStep's value has not been set or is a nonsensical value. for tickets 1258/1263
```

Comment:
Again, the problem with these sorts of hard constants is if var.value ~ 1e-100, then a sliderstep of that value may be appropriate. However 1e-100 probably close to epslion, though, where 1+epsilon == 1 (In IEEE arithmetic), so maybe its appropriate (until it's not - we cannot assume arithemetic will always be IEEE double precision, even though it's 99.9% likely to be so.

Reply:
Epsilon() still seems to be too large for what I had in mind [https://stackoverflow.com/questions/13698927/compare-double-to-zero-using-epsilon](https://stackoverflow.com/questions/13698927/compare-double-to-zero-using-epsilon) 

So, I'm using std::numeric_limits<double>::min() instead, which seems to be a good cutoff value, because most of the time sliderStep is initially either 0 or values like 2.61361e-321.